tests/vm_test.go

```diff
@@ -298,23 +285,14 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 		})
 
 		It("[test_id:3162]should ignore kubernetes and kubevirt annotations to VMI", func() {
-			annotations := map[string]string{
+			vm := libvmi.NewVirtualMachine(libvmifact.NewGuestless(), libvmi.WithRunning())
+			vm.Annotations = map[string]string{
```

Comment:
I believe `vm.Annotations` is of no interest to this test, as it cares about the VMI annotations
(`vm.Spec.Template.ObjectMeta.Annotations`)

Reply:
Good catch. Also, according to @xpivarc's comment here [1]:
```
We are NOT copying annotation from VM.annotation to VMI.
```
[1] https://github.com/kubevirt/kubevirt/pull/3110#issuecomment-626578994

Reply:
I removed it, because the functionality of filtering `kubevirt.io` and `kubernetes.io` annotations was removed in #3110.